catalog/plugin/catalog-plugin-metacard-validation/src/test/java/ddf/catalog/metacard/validation/MetacardValidityFilterPluginTest.java

```diff
@@ -13,9 +13,7 @@
  */
```

Comment:
Not really an issue, just a comment... There are 7 pretty simple to clear out warnings showing up for this file, should we clear those up while in there?